airflow/triggers/temporal.py

```diff
@@ -34,9 +34,13 @@ class DateTimeTrigger(BaseTrigger):
     a few seconds.
 
     The provided datetime MUST be in UTC.
+
+    :param moment: when to yield event
+    :param end_task: whether the trigger should mark the task successful after time condition
+        reached or resume the task after time condition reached.
     """
 
-    def __init__(self, moment: datetime.datetime):
+    def __init__(self, moment: datetime.datetime, *, end_task=False):
```

Comment:
Will we be able to skip resuming simply by setting `next_method` to None?

Reply:
Yes. If the trigger ends the task instance, the "method_name" should be set to 'None'. If the trigger sends execution back to the task, provide the name of the method that should be used.

Reply:
Just want to confirm it. Is `end_task` a convention we should follow if we want to use this feature?

Reply:
In this approach, yes `end_task` needs to be set for the trigger to mark the task successful after the time condition is reached or resume the task after the time condition is reached.